exchanges/bybit/bybit_wrapper.go

```diff
@@ -571,8 +574,13 @@ func (by *Bybit) UpdateAccountInfo(ctx context.Context, assetType asset.Item) (a
 		return info, err
 	}
 	currencyBalance := []account.Balance{}
+	reservedPairs := make(map[string]struct{})
 	for i := range balances.List {
 		for c := range balances.List[i].Coin {
+			_, okay := reservedPairs[balances.List[i].Coin[c].Coin]
+			if okay {
+				continue
+			}
```

Comment:
What a mean is a check down stream in `account.Process()` which will then pick up any future malformed calls. So this check is not necessary.

Reply:
I didn't remove the reservedPairs map instance, not I have